package.json

```diff
@@ -15,7 +15,7 @@
     "@actions/core": "^1.4.0",
     "@actions/github": "^5.0.0",
     "@angular-devkit/build-angular": "13.2.0-rc.1",
-    "@angular/benchpress": "^0.2.2",
+    "@angular/benchpress": "file:./angular-benchpress-v0.2.2.tgz",
```

Comment:
Doesn't this mean we are using a local copy rather than NPM? I am just a little confused on why we need to do this part.

Reply:
This is just temporary until the fix landed in framework. We can hold off for now

Reply:
👍  I ended up seeing that in a review later in my list.  Sounds good